index.html

```diff
@@ -46,69 +52,70 @@ <h1>I'm Benjamin Glad to see you!</h1>
           <img src="photo/logos/Snapshoot-Portfolio.png" alt="">
         </div>
 
-      <div class="hero">
+        <div class="destop-tonic-img">
+          <img src="photo/navs/Snapshoot-Portfolio.jpg" alt="">
+        </div>
+
+        <div class="hero">
           <div class="tonic">
-              <h1>Tonic</h1>
+            <h1>Tonic</h1>
+              <div class="canopy-wrapper">
+                <p>CANOPY</p>
+                  <ul>
+                   <li>Back End Dev</li>
+                     <li>2015</li>
+                  </ul>
+              </div>
```

Comment:
- [ ] Actually, your cards look as it was and I can't see any changes, please, follow the suggestions made by the previous reviewer. Ensure that this section matches the Figma template and you applied the same `font-size`, `colors`, `margin` and `padding`, etc. I know you can do that.

**Your cards:** 

![Screenshot from 2022-07-16 18-56-34](https://user-images.githubusercontent.com/80612925/179364934-c578b96c-9731-48b6-9d1e-6e92cb9af039.png)